docs/md2ipynb.py

```diff
@@ -108,8 +89,6 @@ def convert(path, mode, kernel_name=None, timeout=40 * 60):
 @click.option("--kernel", "-k", help="Name of iPython kernel to use.")
 @click.option("--mode", "-m", default="release")
 def cli(paths, kernel, mode):
-    mode = check_github_event(mode)
```

Comment:
I think we always need to run this script, either with `--mode release` or `--mode skip`.

In the latter case we rename the `.template` files to just `.md` or `.ipynb`.